otelcollector/scripts/setup.sh

```diff
@@ -36,12 +36,13 @@ gem install re2
 
 echo "Installing mdsd..."
 if [ "${ARCH}" != "amd64" ]; then
-  wget https://github.com/microsoft/Docker-Provider/releases/download/mdsd-mac-official-06-13/azure-mdsd_1.19.3-build.master.428_aarch64.rpm
-  sudo tdnf install -y azure-mdsd_1.19.3-build.master.428_aarch64.rpm
+  wget https://github.com/Azure/prometheus-collector/releases/download/azure-mdsd-1.23.3/azure-mdsd_1.23.4-build.master.28_aarch64.rpm
```

Comment:
grace, mdsd is still not part of mariner packages ?

Reply:
It is but the version with the fix for the extra error logging hadn't been published yet as a Mariner package at the time, so mdsd team sent over the RPMs for 1.23.4. 1.23.5 (there's no mariner package for 1.23.4) is now published, but this may reset some testing :) I'm happy to change it and re-test, or I'll add a task to switch it back over for the next release?

Reply:
ok we can pick up next time.